llpc/test/shaderdb/core/OpAccessChain_TestRowMajorBlockVectorExtract_lit.frag

```diff
@@ -1,3 +1,5 @@
+// NOTE: Assertions have been autogenerated by tool/update_llpc_test_checks.py
+// RUN: amdllpc -enable-opaque-pointers=true -gfxip 10.1 -emit-lgc -verify-ir -o - %s | FileCheck -check-prefix=SHADERTEST %s
```

Comment:
This test is much larger because you've re-factored it to use the emit-lgc approach right?

Reply:
Yes. This way I think accidents are more easily caught.

Reply:
Yes, I agree this is better.